src/compas_fab/backends/robot_forward_kinematics.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+class RobotForwardKinematics(object):
```

Comment:
I'm confused here. Where are these classes (`RobotForwardKinematics`, `RobotInverseKinematics`, `RobotPlanCartesianMotion` and `RobotPlanMotion`) used?

Reply:
They aren't.  I added them because of consistency, but they won't remain.